event-handlers/src/main/java/no/sikt/nva/nvi/events/db/DynamoDbUtils.java

```diff
@@ -0,0 +1,53 @@
+package no.sikt.nva.nvi.events.db;
+
+import static java.util.Objects.nonNull;
+import static no.unit.nva.commons.json.JsonUtils.dtoObjectMapper;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent.DynamodbStreamRecord;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+public final class DynamoDbUtils {
+
+    private DynamoDbUtils() {
+    }
+
+    static Map<String, AttributeValue> getImage(DynamodbStreamRecord record) {
+        var image = nonNull(record.getDynamodb().getNewImage())
+                        ? record.getDynamodb().getNewImage()
+                        : record.getDynamodb().getOldImage();
+        return mapToDynamoDbAttributeValue(image);
+    }
+
+    private static Map<String, AttributeValue> mapToDynamoDbAttributeValue(
+        Map<String, com.amazonaws.services.lambda.runtime.events.models.dynamodb.AttributeValue> image) {
```

Comment:
probably exists some alternative to this sdk1 package, but well well lets save something fun for the future :-)

Reply:
or maybe its not ready yet well well, then its all good